spec/integration/fielded_notice_search_spec.rb

```diff
@@ -265,10 +265,8 @@
           search_on_page.add_fielded_search_for(field, 'test')
         end
 
-        sleep 0.2
-
         search_on_page.within_fielded_searches do
-          expect(page).to have_no_css('#duplicate-field')
+          expect(page).to have_no_css('#duplicate-field', wait: 5)
```

Comment:
Nice. This test has really been annoying. It's too bad the fix is to slow it down, but if we can confine all the slow tests to our integration suite, we're way better off than we used to be.